### ISSUE TITLE
Reduce declare of global variables

### ISSUE THREAD

**Issue by elong0527** [Merck/boxly#12]
Why would we have so many global variables? Better to check what is absolutely required. 

```
utils::globalVariables(c(
  "ANL01FL", "boxly_adlb", "CHG", "Freq", "group", "label", "outdata", "outlier",
  "param", "PARAMCD", "text", "TRT01A", "TRTA", "x", "AVISITN"
))
```

**Comment by elong0527** [comment]
Removed `global.R` and get information below. I will handle it before CRAN submission. 

```
boxly: no visible binding for global variable ‘x’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/boxly.R:144-155)
boxly: no visible binding for global variable ‘outlier’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/boxly.R:144-155)
boxly: no visible binding for global variable ‘group’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/boxly.R:144-155)
boxly: no visible binding for global variable ‘text’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/boxly.R:144-155)
meta_boxly: no visible binding for global variable ‘boxly_adsl’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/meta_boxly.R:28)
meta_boxly: no visible binding for global variable ‘boxly_adlb’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/meta_boxly.R:29)
meta_boxly: no visible binding for global variable ‘ANL01FL’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/meta_boxly.R:41-87)
meta_boxly: no visible binding for global variable ‘AVISITN’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/meta_boxly.R:41-87)
meta_boxly: no visible binding for global variable ‘CHG’
  (/home/runner/work/boxly/boxly/check/boxly.Rcheck/00_pkg_src/boxly/R/meta_boxly.R:41-87)
meta_boxly: no visible binding for global variable ‘PARAMCD’
```

**Comment by elong0527** [comment]
Addressed in #18
